update-sample-distribution-analysis (1/N)

AntoniaChroni opened the pull request:
<!--Hi there, thanks for your contribution! Please take a moment to fill out this template to facilitate the review of your pull request.-->

### Purpose/implementation Section

#### What scientific question is your analysis addressing?

Here, we explore longitudinal data in the PBTA cohort.

#### What was your approach?



#### What GitHub issue does your pull request address?

Update input and output files.

### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.

#### Which areas should receive a particularly close look?

Please review the script logic. 

#### Is there anything that you want to discuss further?


#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?

Yes.

### Results

#### What types of results are included (e.g., table, figure)?

There are three tables generated for cases with matched longitudinal samples:
- `genomic_assays_matched_time_points.tsv`
- `paired_assays_matched_time_points.tsv`
- `transcriptomic_assays_matched_time_points.tsv`

#### What is your summary of the results?

There are 2248 patient cases in total with longitudinal information. In cases with only genomic assays, there are 1997 patient cases with 119 having matched longitudinal samples. In cases with only transcriptomic assays, there are 1839 patient cases with 106 having matched longitudinal samples. In cases with both paired genomic and transcriptomic assays, there are 1541 patient cases with 70 having matched longitudinal samples.

The five cancer types with the highest number of tumor samples (>100) therein are: Low-grade glioma, Medulloblastoma, Diffuse midline glioma, High-grade glioma, and Ependymoma. 

#### Reproducibility Checklist

<!-- Check all those that apply or remove this section if it is not applicable.-->

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.

#### Documentation Checklist

<!-- Please review https://github.com/d3b-center/pbta-tumor-evolution#documenting-your-analysis -->

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

<!-- IF YOUR PULL REQUEST IS A DATA RELEASE, PLEASE REMOVE THE [HTML COMMENT TAG](https://html.com/tags/comment-tag/) FROM THE SECTION BELOW AND COMPLETE THE CHECKLIST-->

<!-- 
### Data Release Checklist
- [x] Is the table in doc/data-file-descriptions.md up to date?
- [ ] Is doc/data-format.md up to date?
- [ ] Is doc/release-notes.md up to date?
- [x] Is download-data.sh up to date?
- [x] Was download-data.sh tested and did it complete without error?
-->

Commit:
Module create-pptc-pdx-corplots

Commit:
removed not necessary files

Commit:
script 1-process-data_VAF

to estimate VAFs and save files per patient

Commit:
remove plots, update readme

Commit:
remove archive

Commit:
remove plots dir

Commit:
edit name in analysis_dir

Commit:
edit name 1-process-data_VAF

Commit:
add part for only deceased samples

Commit:
edit deceased samples

Commit:
save maf_files_Deceased.tsv file and minor edits

Commit:
Merge branch 'main' into create-corplots

Commit:
add libraries

Commit:
added violin_plot & Stacked_barplot

Commit:
added tmb file

Commit:
edit Stacked_barplot

Commit:
added code for corplots

Commit:
Update README.md

Commit:
fix typo in readme

AntoniaChroni committed:
Update analyses/create-corplots/1-process-data_VAF.Rmd

Co-authored-by: Jo Lynne Rokita <jharenza@gmail.com>

AntoniaChroni committed:
Update analyses/create-corplots/1-process-data_VAF.Rmd

Co-authored-by: Jo Lynne Rokita <jharenza@gmail.com>

Commit:
edits on load and process data

Commit:
remove input files not necessary

-pbta
- genomic_df_filter

Commit:
clean pbta

Commit:
edits in reading maf file

Commit:
update script and README.md files

Commit:
add bash file run-create-corplots.sh

Commit:
add scripts

jharenza commented:
>Please also notice cases where we have the same time point pairs, e.g. Diagnosis-Diagnosis, or Diagnosis-Recurrence-Recurrence. This can be seen in the generated tumor_descriptor_df_Diagnosis, see Lines 181-201.

Some options for this could be:
- if WGS and WXS, select only one
- Use the independent specimen files to select a specimen or randomly sample a specimen if multiple per time point
- Looking at each sample in your plots (so you'd have Diagnosis-bsid1 + Recurrence and Diagnosis-basid2 + recurrence)
- Combining the data at the same time point and uniquing the variants  - I might favor this one _solely_ for correlation plots, as they describe tumor descriptor (time point) rather than site of sample, which may be different when we have two different samples at Diagnosis

Commit:
updates

Commit:
Revert "updates"

This reverts commit 2dfbc77e736169973c4e97e6b083de317b6c9a4a.

Commit:
Merge remote-tracking branch 'origin/update-sample-distribution-analysis' into update-sample-distribution-analysis

Commit:
remove files for corplots module

Commit:
Clean library and set-dir chunk

Commit:
Merge load and process data chunk

Commit:
Clean cohort PBTA chunk

Commit:
Edit cohort PBTA chunk to filter out duplicates

# The file contains duplicates because some samples might have:
# (1) both WGS and WXS, and/or (2) multiple biospecimen samples.
# We will address these issues by creating a unique identifier per each sample and assay (match_id_DNA).

Commit:
Clean text

Commit:
Fix experimental_strategy_sum column

Commit:
Fix typo

Commit:
Add matched_samples_transcriptomic and matched_samples_paired

Commit:
Remove duplicate events - paired assays chunk

Commit:
Clean text

Commit:
Clean code based on Lint

Commit:
Update README.md

Commit:
Fix typo

Commit:
Clean text in notebook and readme file

Commit:
Clean text more in the notebook

Commit:
Clean code in the notebook

Commit:
Update 01-tumor-descriptor-and-assay-count-plot.nb.html

Commit:
Add Results section

Commit:
Fix typo

Commit:
Edit name in the script file

Remove word "plot"

Commit:
Update name of script in the README file

Commit:
Remove author name and mutate experimental_strategy

Commit:
Remove "Second Malignancy" from filter for paired assays

Commit:
Run Rmd in docker and update html

Commit:
Edit #cases in the matched longitudinal samples

The number changed after I removed the cases with "Second malignancy".

rjcorb commented on analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd:
```diff
@@ -0,0 +1,386 @@
+                                      TRUE ~ tumor_descriptor),
+         match_id = paste(tumor_descriptor, sample_id, sep = "_"),
+         DNA = case_when(grepl("WGS", experimental_strategy) ~ "DNA",
+                         grepl("WXS", experimental_strategy) ~ "DNA", 
```
```suggestion
         DNA = case_when(grepl("WGS|WXS", experimental_strategy) ~ "DNA",
```
To cut down on # lines

rjcorb commented on analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd:
```diff
@@ -0,0 +1,386 @@
+  mutate(tumor_descriptor = case_when(grepl("C4106355-Tumor", sample_id) ~ "Recurrence",
+                                      grepl("C4113612-Tumor", sample_id) ~ "Diagnosis",
+                                      grepl("Primary Tumor", tumor_descriptor) ~ "Diagnosis",
+                                      grepl("Initial CNS Tumor", tumor_descriptor) ~ "Diagnosis",
```
```suggestion
                                      grepl("Primary Tumor|Initial CNS Tumor", tumor_descriptor) ~ "Diagnosis",
```

rjcorb commented on analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd:
````diff
@@ -0,0 +1,386 @@
+Let's select all patient samples containing both genomic and transcriptomic assays.
+```{r assays, echo=TRUE}
+assays <- pbta %>%
+  select(Kids_First_Participant_ID, tumor_descriptor, match_id, experimental_strategy, cancer_group, DNA, match_id_DNA) %>%
````
```suggestion
  dplyr::select(Kids_First_Participant_ID, tumor_descriptor, match_id, experimental_strategy, cancer_group, DNA, match_id_DNA) %>%
```
I've run into some issues in the past when just using `select()` on its own, so I've started always specifying dplyr package first

rjcorb commented on analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd:
````diff
@@ -0,0 +1,386 @@
+We’ll look at the breakdown of the tumor_descriptor column, separating the genomic from transcriptomic assays.
+```{r}
+tumor_descriptor_df <- paired_assays_all %>%
+  arrange(Kids_First_Participant_ID)
````
as far as I can tell, `tumor_descriptor_df` is identical to `paired_assays_all`, so I think you can just use `paired_assays_all` for all downstream code.

AntoniaChroni replied:
Great catch! This has now been addressed, see 87c7c8120bfd7eee57dee0456287c132d06da94f

rjcorb commented on analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd:
````diff
@@ -0,0 +1,386 @@
+genomic_assays
+```
+
+```{r}
+genomic_assays %>%
+  group_by(Kids_First_Participant_ID, descriptors, experimental_strategy) %>%
+  tally() %>%
+  arrange(desc(n)) %>%
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
+```
````
I think this is too long of a table to be included here.

AntoniaChroni replied:
I agree that this and some other tables are too long and overwhelming to be included in the notebook.
Let's see what @jharenza thinks of how to address this issue.

AntoniaChroni replied:
This has been removed now, see 9e5671c9b993d7a0e0cecb7e356da317bc196f5c

AntoniaChroni replied:
I also removed it for transcriptomic_assays, 9de334a471330e14dbac673500791dd32e15805b

rjcorb commented on analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd:
````diff
@@ -0,0 +1,386 @@
+  fontsize(size = 12, part = "all")
+```
+
+```{r}
+genomic_assays_matched <- genomic_assays %>%
````
Can you include some documentation for this chunk?

AntoniaChroni replied:
Documentation is now included, see here a38aa5c67c04bab1398b3f04577b15434273e7d6

rjcorb commented on analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd:
```diff
@@ -0,0 +1,386 @@
+                            "Deceased, Progressive", "Recurrence, Second Malignancy", "Progressive, Recurrence", 
+                            "Diagnosis, Second Malignancy", "Deceased, Progressive, Recurrence", "Diagnosis, Progressive, Recurrence",
+                            "Deceased, Recurrence", "Deceased, Diagnosis, Progressive, Recurrence", "Deceased, Diagnosis, Recurrence",
+                            "Deceased, Diagnosis, Progressive", "Diagnosis, Unavailable")) %>%
```
A suggestion -- if the purpose of these lines is to filter for patients with genomic data at different time points, you could do `grepl(", " descriptors)` instead of manually entering all the combinations. That way the code will be robust in the future in case other timepoints combos of genomic data become available

AntoniaChroni replied:
There are patient cases with only one `tumor_descriptor ` and I wanted to get rid of those. But to address the issue of possible other combos showing in the future I change the filter, see dfe29f1351233fe13e19e4dab23e43577a9d399c

rjcorb commented on analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd:
```diff
@@ -0,0 +1,386 @@
+  tally() %>%
+  arrange(desc(n)) %>%
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
```
I think, since this is exported in the code chunk above, it may not need to be summarized printed here given how long of a table it is.

AntoniaChroni replied:
Addressed here a616809ef186ee4e78ca49bf6b19f30c899bba57

rjcorb commented on analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd:
````diff
@@ -0,0 +1,386 @@
+  fontsize(size = 12, part = "all")
+```
+
+```{r}
+transcriptomic_assays  %>%
+  group_by(Kids_First_Participant_ID, descriptors, experimental_strategy) %>%
+  tally() %>%
+  arrange(desc(n)) %>% 
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
+```
````
Same as tables above, I think this is too long for the html file.

AntoniaChroni replied:
@jharenza What do you think?

AntoniaChroni replied:
I removed it for `transcriptomic_assays`, https://github.com/d3b-center/pbta-tumor-evolution/commit/9de334a471330e14dbac673500791dd32e15805b and `transcriptomic_assays_matched`, 89a0e212dc2ed93453bf7f44910bf2c0aef47155

rjcorb commented on analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd:
````diff
@@ -0,0 +1,386 @@
+```{r}
+transcriptomic_assays_matched <- transcriptomic_assays %>%
+  filter(descriptors %in% c("Diagnosis, Recurrence", "Deceased, Diagnosis", "Diagnosis, Progressive",
+                            "Deceased, Recurrence", "Progressive, Recurrence", "Deceased, Progressive", "Deceased, Diagnosis, Progressive")) %>%
````
same suggestion here as for `genomic_assays_matched` `filter(grepl(", ", descriptors)

AntoniaChroni replied:
Addressed here feb7870ffa9f34aa8d80e381969ddc669d2258b3

rjcorb commented on analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd:
```diff
@@ -0,0 +1,386 @@
+transcriptomic_assays_matched <- transcriptomic_assays %>%
+  filter(descriptors %in% c("Diagnosis, Recurrence", "Deceased, Diagnosis", "Diagnosis, Progressive",
+                            "Deceased, Recurrence", "Progressive, Recurrence", "Deceased, Progressive", "Deceased, Diagnosis, Progressive")) %>%
+  mutate(experimental_strategy = gsub(";WGS|WXS","", experimental_strategy)) %>%
```
```suggestion
  mutate(experimental_strategy = gsub(";WGS|;WXS","", experimental_strategy)) %>%
```

added a `;` before `WXS` because the original code resulted in one row where `experimental strategy == "RNA-seq;"`

AntoniaChroni replied:
I don't see that on my end. Both codes result in the same table. What to do in this situation? @jharenza

rjcorb replied:
Here is what I see when running the original code: 

```
> table(transcriptomic_assays_matched$experimental_strategy)

 RNA-Seq RNA-Seq; 
     157        1 
```

versus making the suggested change: 

```
> table(transcriptomic_assays_matched$experimental_strategy)

RNA-Seq 
    158 
```

rjcorb commented on analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd:
```diff
@@ -0,0 +1,386 @@
+  mutate(experimental_strategy = gsub(";WGS|WXS","", experimental_strategy)) %>%
+  write_tsv(file.path(results_dir, "transcriptomic_assays_matched_time_points.tsv"))
+
+transcriptomic_assays_matched %>%
+  group_by(Kids_First_Participant_ID, descriptors, experimental_strategy) %>%
+  tally() %>%
+  arrange(desc(n)) %>%
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
```
Does it make sense that some of these patients have two separate RNA-seq assays at both of the timepoints listed?

AntoniaChroni replied:
Each timepoint (`tumor_descriptor`) has its own RNA-Seq assay. To avoid confusion we decided to show this as a single entry `RNA-Seq` instead of `RNA-Seq; RNA-Seq`. 
You should be seeing one RNA-Seq assay as shown below. Please let me know if this answers your question, or you meant something else.

![Screenshot 2023-08-11 at 8 14 13 PM](https://github.com/d3b-center/pbta-tumor-evolution/assets/46328680/fa7d6e2e-b152-41d5-9ef6-73dd29a3bcfd)

rjcorb commented on analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd:
````diff
@@ -0,0 +1,386 @@
+  fontsize(size = 12, part = "all")
+```
+
+```{r}
+paired_assays %>%
+  group_by(Kids_First_Participant_ID, descriptors, experimental_strategy) %>%
+  tally() %>%
+  arrange(desc(n)) %>%
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
+```
````
Remove due to length

AntoniaChroni replied:
@jharenza

AntoniaChroni replied:
Removed see here 51ae26716f00de571b1d9b41f3868677e8679966

rjcorb commented on analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd:
```diff
@@ -0,0 +1,386 @@
+                            "Recurrence, Second Malignancy", "Progressive, Recurrence", "Diagnosis, Second Malignancy", 
+                            "Deceased, Progressive, Recurrence", "Diagnosis, Progressive, Recurrence", "Deceased, Recurrence", "Deceased, Diagnosis, Recurrence",
+                            "Diagnosis, Unavailable")) %>%
+  write_tsv(file.path(results_dir, "paired_assays_matched_time_points.tsv"))
```
see suggestion for filtering other matched dfs

AntoniaChroni replied:
See here 0f760e29e8088fce3acf30ad1d823c47a9fb9f24

rjcorb commented on analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd:
```diff
@@ -0,0 +1,386 @@
+                            "Diagnosis, Unavailable")) %>%
+  write_tsv(file.path(results_dir, "paired_assays_matched_time_points.tsv"))
+
+paired_assays_matched %>%
+  group_by(Kids_First_Participant_ID, descriptors, experimental_strategy) %>%
+  tally() %>%
+  arrange(desc(n)) %>%
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
```
remove due to table length

AntoniaChroni replied:
@jharenza

AntoniaChroni replied:
Removed see here 0d3728ccb96f6604021a11f44c855375c93c675d

rjcorb commented on analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd:
````diff
@@ -0,0 +1,386 @@
+# Genomic assays and cancer_group
+Let us investigate the number of tumor samples with genomic assays per cancer_group.
+```{r}
+list <- genomic_assays %>%
+  count(cancer_group) %>%
+  arrange(desc(n))
````
One thing to keep in mind is that value of `cancer_group` group varies based on number of time points, even when patients belong to the same cancer group. For example, an HGG patients with one sample will have `High-grade glioma`, but an HGG patient with two time points will be `High-grade glioma, High-grade glioma` so this isn't really getting a breakdown of # patients by cancer group. This could be resolved by collapsing the `cancer_group` column to unique values when creating `genomic_assays`, but I'm not sure if that's what you want to do.

AntoniaChroni replied:
Then maybe use `harmonized` or `broad_histology` instead will be more appropriate in this case?
@jharenza

rjcorb commented on analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd:
````diff
@@ -0,0 +1,386 @@
+
+We will look into the first five cancer groups with the highest number of tumor samples: Low-grade glioma, Medulloblastoma, Diffuse midline glioma, High-grade glioma, and Ependymoma. There are cases that the second time point has a different cancer type. Let us summarize this to ensure we capture all tumor samples regardless of diagnosis changes at a later stage of the disease.We will only do thi ...
+
+```{r}
+genomic_assays_matched <- genomic_assays_matched %>% 
+  mutate(cancer_group_broad = case_when(grepl("Low-grade glioma", cancer_group) ~ "Low-grade glioma",
+                                        grepl("Medulloblastoma", cancer_group) ~ "Medulloblastoma", 
+                                        grepl("High-grade glioma", cancer_group) ~ "High-grade glioma",
+                                        grepl("Ependymoma", cancer_group) ~ "Ependymoma", 
+                                        grepl("Diffuse midline glioma", cancer_group) ~ "Diffuse midline glioma"))
+```
````
Good solution here for resolving the issue I just mentioned :). I did notice that there is one patient with `cancer_group == "High-grade glioma, Low-grade glioma"`, and as of right now that would be assigned to the LGG group. It might be better to prioritize HGG assignment over LGG for cases like this.

AntoniaChroni replied:
Great catch! I didn't realize that would have been replaced. I switched the order, see here: 0704391c08132a297251dc30eaa9b51ef958e3ab

rjcorb reviewed (commented):
This is looking good! I appreciated the thorough documentation to all of your code, it helped a lot with interpreting steps of the analysis. I made a few minor suggestions that might help this code be more robust in the future. One general comment I had, that I made in a few specific code chunks, is that some of the printed tables with separate rows per patient are a bit cumbersome to include in the html file. I would recommend just including the more summarized tables by `tumor_descriptor`. Let me know if you have any specific questions. I don't need to review again.

AntoniaChroni committed:
Update analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd

Co-authored-by: Ryan Corbett <corbettr@chop.edu>

AntoniaChroni committed:
Update analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd

Co-authored-by: Ryan Corbett <corbettr@chop.edu>

AntoniaChroni committed:
Update analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd

Co-authored-by: Ryan Corbett <corbettr@chop.edu>

Commit:
Replace tumor_descriptor_df with paired_assays_all

Commit:
Add documentation for genomic_assays_matched

Commit:
Filter single tumor_descriptor from genomic assays

Commit:
Remove flextable for genomic_assays_matched

Commit:
Filter single tumor_descriptor from transcriptomic assays

Commit:
Filter single tumor_descriptor from paired_assays_matched

Commit:
Prioritize High-grade glioma over Low-grade glioma

naqvia commented on analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd:
```diff
@@ -0,0 +1,375 @@
+         match_id_DNA = paste(Kids_First_Participant_ID, tumor_descriptor, DNA, sep = "_")) %>%
+  distinct(match_id_DNA, .keep_all = TRUE) # filter out
+
+print(pbta %>% count(tumor_descriptor))
```
this is producing an error for me. alternatively you can try `print (count(pbta$tumor_descriptor_final))`

AntoniaChroni replied:
This code should be for the `tumor_descriptor`, there is no `tumor_descriptor_final`.
I tried it anyway, but it produces an error for me (see below). Can you please confirm that you run this module via Docker?

> print (count(pbta$tumor_descriptor))
Error in UseMethod("count") : 
  no applicable method for 'count' applied to an object of class "character"

naqvia commented on analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd:
````diff
@@ -0,0 +1,375 @@
+# Diagnosis vs. any other timepoint (Recurrence, Progressive etc)
+We’ll look at the breakdown of the tumor_descriptor column, separating the genomic from transcriptomic assays.
+```{r}
+paired_assays_all %>%
+  arrange(Kids_First_Participant_ID) %>%
+  group_by(tumor_descriptor, experimental_strategy_sum) %>%
+  tally() %>%
+  arrange(desc(n)) %>% 
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
````
Getting the following error: 
```
Error in eval(quote(list(...)), env) :
object 'Kids_First_Participant_ID' not found
```

You are losing the info from the above code where you use `summarise`: 
```
assays %>%
  group_by(Kids_First_Participant_ID, match_id) %>%
  summarise(experimental_strategy = str_c(experimental_strategy, collapse = ";"))
```
It is because the `summarise` call creates a new df (https://dplyr.tidyverse.org/reference/summarise.html), so you are only left with two columns: 
```
> names(paired_assays_all)
[1] "experimental_strategy"     "experimental_strategy_sum"
```
This also causes downstream code to break. I suggest you separate out the `summarise` portion (ie notusing the pipe) and add it later if needed.

AntoniaChroni replied:
I am creating the main df and then when I create the flextables, I do that separate so I won't miss the information needed later on in the notebook. For the `paired_assays_all`, the df is created in Lines 96-101. Can you please confirm that this is via Docker?

naqvia reviewed (changes requested):
The error produced bc of the use of the `summarise` function call is breaking the code (see specific comments). When I commented out the lines using the `summarise` function (it is also used in another place in the `Genomic assays` code block), the script runs as expected! I made some suggestons, and once addressed, I think we'd be in great shape!

Commit:
Remove flextable for genomic_assays per Kids_First_Participant_ID

Commit:
Remove flextable for transcriptomic_assays per Kids_First_Participant_ID

Commit:
Remove flextable for transcriptomic_assays_matched per Kids_First_Participant_ID

Commit:
Remove flextable for paired_assays per Kids_First_Participant_ID

Commit:
Remove flextable for paired_assays_matched per Kids_First_Participant_ID

Commit:
Update html report after integrating all changes

rjcorb reviewed (approved):
Looks good to me! I made an additional comment on line 127 regarding a suggested change, but I don't need to review again.

AntoniaChroni commented:
> Looks good to me! I made an additional comment on line 127 regarding a suggested change, but I don't need to review again.

That was meant for `Line 227`, correct? @rjcorb 

Resolved here 65ff29f060e842a0022e0afd153cc5a325bb92ed

AntoniaChroni committed:
Update analyses/sample-distribution-analysis/01-tumor-descriptor-and-assay-count.Rmd

Co-authored-by: Ryan Corbett <corbettr@chop.edu>

Commit:
Add write_tsv to save pbta cohort because we'll use through the project a lot

Commit:
Add pbta file and html after run Rmd via Docker

Commit:
Merge branch 'main' into update-sample-distribution-analysis

AntoniaChroni commented:
This script has been updated based on suggestions by @rjcorb.
@jharenza and I decided to save also the `pbta df` in `results_dir`, because this file will be needed in other modules across the project. See e14cbd35c900d6273007ef0a2cb2ea0c5202ca8d

Please run this one last time to ensure that all files are included in the `results_dir`. @rjcorb @naqvia

Commit:
Update README file for including pbta.tsv in results.dir

rjcorb commented:
Looks good! Can confirm `pbta-tsv` is saved to `results_dir`

naqvia reviewed (approved):
Runs as expected and saves desired results. I would just make sure to update the readme to include the updated script name!

Commit:
Update README file with the updated script name
